app/src/main/java/com/khanhhuy/movie_db/adapter/CategoryAdapter.java

```diff
@@ -0,0 +1,57 @@
+package com.khanhhuy.movie_db.adapter;
+
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.khanhhuy.movie_db.BR;
+import com.khanhhuy.movie_db.R;
+import com.khanhhuy.movie_db.databinding.ItemListFragmentHomeBinding;
+import com.khanhhuy.movie_db.model.Category;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yeu_thuong on 11/5/2017.
+ */
+
+public class CategoryAdapter extends RecyclerView.Adapter<CategoryAdapter.CategoryViewHolder> {
+    private List<Category> mCategories;
+
+    public CategoryAdapter(List<Category> categories){
+        mCategories = categories;
+    }
+
+    @Override
+    public CategoryViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_list_fragment_home, parent, false);
+        return new CategoryViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(CategoryViewHolder holder, int position) {
+        Category category = mCategories.get(position);
+        holder.bindData(category);
+    }
+
+    @Override
+    public int getItemCount() {
+        return (mCategories == null)? 0 : mCategories.size();
+    }
+
+    public static class CategoryViewHolder extends RecyclerView.ViewHolder{
+        ItemListFragmentHomeBinding listFragmentHomeBinding;
```

Comment:
check convention 
32 Java style rules

2.1 Java Style Rules

Field names need to be defined at first and follow this conventions.

Non-public, non-static field names start with m.
Static field names start with s.
Other fields start with a lower case letter.
Public static final fields (constants) are ALL_CAPS_WITH_UNDERSCORES.